docs/commerce/shipping_management/work_with_shipments.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Work with shipments and view shipment details.
+edition: commerce
+---
+
+# Work with shipments
+
+In Ibexa DXP, on the **Shipments** screen, you can view a list of shipments and modify their statuses.
+You can access shipments for your own orders or all the shipments that exist in the system, depending on your permissions.
+
+![Shipment list](shipment_list.png)
+
+## View shipment status
```

Comment:
I would go a different path - call the section "Filter shipments in shipment list" and move statuses to where they are discussed among other filters.

But this is just a suggestion.